### ISSUE TITLE
question about godep save

### ISSUE THREAD

**Issue by prateekgogia** [tools/godep#371]
HI,

I have question about Godep save. I have a project X currently not using godep. I want to import project Y, which heavily relies on Godeps. I have added Godeps in my project X with project Y  and its dependencies. Everything works fine, I am able to build doing "godep go build". 
The problem comes when I am trying to import a new package to my project. In this case once I build everything, the read me says to run godep save or godep save ./... . Now when I am trying to do this, save errors out saying  - 

godep: cannot find package "github.com/docker/docker/pkg/units" in any of: 

           /usr/local/go/src/github.com/.....  (from $GOROOT)
           ......./../src/github.com/.../../...  (from $GOPATH)

why is this trying to get the package again, if I just do godep save github.com/mynewpackageimport/ - it removes everything form Godep.json and _workspace folder. 
Even when it is trying to get these packages why does it fails, because it should be using the version in Godep.json file to fetch this.

In short, need help saving a new package into the Godeps.

godep version
godep v32 (linux/amd64/go1.5.1)

uname -a
Linux build 4.2.0-16-generic #19-Ubuntu SMP Thu Oct 8 15:35:06 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

**Comment by freeformz** [comment]
The package argument that you pass to `godep save` is the package(s) that you want to analyze for vendoring, not the package that you want to save. Put another way it's the packages who's dependencies you want save. Does that help?

**Comment by freeformz** [comment]
Closing issues before archiving. Thank You\!
